src/components/Privacy/createComponent.js

```diff
@@ -25,9 +25,13 @@ export default ({
 
   const readCookieIfQueueEmpty = () => {
     if (taskQueue.length === 0) {
+      const storedConsent = readStoredConsent();
```

Comment:
Is this supposed to be part of this commit?

Reply:
I sneaked this in there because I noticed a warning in the console when we were going through the demos :)  (@jonsnyder to confirm the fix)

Reply:
👍